src/functions.date.js

```diff
@@ -108,27 +108,32 @@ for (let unit in s) {
 	}, {multiValued: true});
 }
 
-_.duration = $.extend(function($this, ms, terms = 1) {
+_.duration = $.extend(function($this, ms, terms) {
 	if (arguments.length === 1) {
 		[ms, $this] = [$this, null];
 	}
 	
+	let wantString = terms === undefined ? true : false;
```

Comment:
`x? true : false` is an antipattern: what you're essentially doing is converting `x` to boolean, so you can express it explicitly as `Boolean(x)` (or `!!x`, though that's less readable). In this case `terms === undefined` is *already* a boolean, so you can just do `let wantString = terms === undefined;`. 

Also, I don't think it's a good idea to have the logic about what to return here, I think it would be easier to read closer to the actual return statement. You can also use `arguments.length` to avoid having to store a variable about whether to return a string or not.

So it can be as succinct as `return arguments.length < 2? ret[0] : ret;` (though it would need to account for no `$this` being provided so it will be a little more complex than that).